compiler/rustc_middle/src/query/mod.rs

```diff
@@ -93,6 +93,10 @@ rustc_queries! {
         desc { |tcx| "computing the optional const parameter of `{}`", tcx.def_path_str(key.to_def_id()) }
     }
 
+    query const_param_default(param: DefId) -> &'tcx rustc_middle::ty::consts::Const<'tcx> {
```

Comment:
Please add some doc comments explaining what you get and what the DefId refers to

Reply:
also small nit here:
```suggestion
    query const_param_default(param: DefId) -> &'tcx ty::Const<'tcx> {
```